CONTRIBUTING.md

```diff
@@ -2,32 +2,97 @@
 
 ## Project Philosophy
 
-Project Cognoma is an open source community-based project. Beyond our immediate scientific aspirations, we want to engage as many individuals as possible in doing cool work. Project Cognoma is a place to learn and grow as a community. So don't be shy. We want your contribution. Given the nature of open source development, not all contributions can be included … but all will be appreciated and help the project bloom.
+Project Cognoma is an open source community-based project. Beyond our immediate scientific aspirations, we want to engage as many individuals as possible in doing cool work. Project Cognoma is a place to learn and grow as a community. So don't be shy. Cognoma wants your contribution!
```

Comment:
There are too solutions:

1. Wrap lines with newlines at 80 characters.
2. One sentence per line

I like 2 because you can comment and diff on a per sentence level. @gwaygenomics you okay with option 2?